src/TaskRunner/Commands/InstallCommands.php

```diff
@@ -76,102 +59,32 @@ public function clone() {
    * - Check current status of configuration
    * - Import configuration from datastore into activestore.
    *
-   * @command toolkit:install-clean
+   * @command toolkit:install-clone
    */
-  public function clean() {
-    $this
-      ->taskExecStack()
+  public function clone() {
```

Comment:
First line of this dockblock says `Install clean website`, should be clone.

Reply:
Done.